x-pack/legacy/plugins/license_management/public/register_route.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { App } from 'src/core/public';
+
+/* Legacy Imports */
+import { npSetup, npStart } from 'ui/new_platform';
+import routes from 'ui/routes';
+import { I18nContext } from 'ui/i18n';
```

Comment:
I believe `I18nContext` is available via the NP. See https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/index_management/public/app/index.tsx#L18 for an example.

Reply:
Thanks for pointing this out! Will update!